### DISCUSSION TITLE
Error:  matchVariant is not a function

### DISCUSSION THREAD

**Discussion by sergitejada** [tailwindlabs/tailwindcss#11893] [Help]
created a new project with tailwind and container query
![image](https://github.com/tailwindlabs/tailwindcss/assets/49126831/866cac11-850b-4a3d-a356-d8fe7fa9a376)
Npm version 8.5.4
node version 18.16.1
Using in rails framework in a windows machine

To start tailwind i use:
web: rails server -p 3000
css: rails tailwindcss:watch

![image](https://github.com/tailwindlabs/tailwindcss/assets/49126831/6abfb4d5-470c-4d7e-bb2d-1776d08fe42f)

**Comment by wongjn** [comment]
I'm not familiar with the Rails framework, but I can see that `@tailwindcss/container-queries` is located at `C:\code\guardian\node_modules\@tailwindcss/container-queries` but Tailwind seems to be located at `C:\snapshot\tailwindcss`. Consider checking this Tailwind CSS is version `v3.2.0` or higher.
